workbench/src/renderer/components/PluginModal/index.jsx

```diff
@@ -63,34 +63,29 @@ export default function PluginModal(props) {
     <Modal.Body>
       <Form>
         <Form.Group className="mb-3">
-          <Form.Label htmlFor="url">Add a plugin</Form.Label>
+          <Form.Label htmlFor="url">{t('Add a plugin')}</Form.Label>
           <Form.Control
             id="url"
-            name="url"
-            type="text"
             placeholder={t('Enter Git URL')}
             onChange={(event) => setURL(event.currentTarget.value)}
           />
           <Form.Text className="text-muted">
             {t('This may take several minutes')}
           </Form.Text>
           <Button
-            name="submit"
             disabled={loading}
             className="mt-2"
-            onClick={handleSubmit}
+            onClick={addPlugin}
           >
             {t('Add')}
           </Button>
         </Form.Group>
         <hr />
         <Form.Group className="mb-3">
-          <Form.Label htmlFor="url">Remove a plugin</Form.Label>
+          <Form.Label htmlFor="plugin-select">{t('Remove a plugin')}</Form.Label>
           <Form.Control
             id="plugin-select"
             as="select"
-            name="plugin"
-            type="text"
```

Comment:
(Just to be clear, removing `type="text"` from this form control was correct, because this one is rendered as a `<select>`, not an `<input>`.)